content/en/logs/log_collection/nodejs.md

```diff
@@ -128,6 +128,53 @@ logs:
     sourcecategory: sourcecode
 ```
 
+## Aggentless Logging
+
+It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
+
+{{< tabs >}}
+{{% tab "Winston 3.0" %}}
+
+You can use [winston HTTP transport][1] to send your logs directly through the datadog API.
+In your bootstrap file or somewhere in your code, declare the logger as follow:
+
+```js
+
+const { createLogger, format, transports } = require('winston');
+
+const httpTransportOptions = {
+  host: 'http-intake.logs.datadoghq.com',
+  path: '/v1/input/{put your API key here}',
```

Comment:
We should recommend to set the source as `nodejs` and maybe service as their application name.

This can be added as query parameter `/v1/input/<APIKEY>?ddsource=nodejs&service=<APPLICATION_NAME>`

What do you think?

Reply:
looks better indeed! 👍

Reply:
added